packages/api-client/lib/openapi/api.ts

```diff
@@ -9372,6 +9372,7 @@ export const TasksApiAxiosParamCreator = function (configuration?: Configuration
      * @param {string | null} [category] comma separated list of task categories
      * @param {string | null} [assignedTo] comma separated list of assigned robot names
      * @param {string | null} [status] comma separated list of statuses
+     * @param {string | null} [label] comma separated list of labels, each item must be in the form &lt;key&gt;&#x3D;&lt;value&gt;, multiple items will filter tasks with all the labels
```

Comment:
Looks like this needs to be updated on the dashboard package's usage

Reply:
thanks for the note!